lib/src/client.dart

```diff
@@ -120,10 +121,13 @@ class Client extends http.BaseClient {
   /// This will also automatically refresh this client's [Credentials] before
   /// sending the request if necessary.
   @override
-  Future<http.StreamedResponse> send(http.BaseRequest request) async {
+  Future<http.StreamedResponse> send(
+    http.BaseRequest request, {
+    RequestProperties? requestProperties,
```

Comment:
Je tohle OK? Mám za to , že Dart neumí přetížit metodu o parametr navíc

Reply:
A navíc, akceptuj čistě ITrackable objekt

Reply:
zruseno, pouziji se jiz pripravene hlavicky